### DISCUSSION TITLE
coder server seemed abnormal?

### DISCUSSION THREAD

**Discussion by femto** [coder/coder#16025] [General]
Hello, just want to try out coder,
```coder server```
when connects to https://ma2c5sn5emhao.pit-1.try.coder.app
<img width="482" alt="Image" src="https://github.com/user-attachments/assets/dde3c3ea-9ae7-4df4-aec8-8bd9c5e6ad10" />

Do I need my machine to be in public domain?

and then http://127.0.0.1:3000/ , seemed agent not working? agent not connected?
<img width="1240" alt="Image" src="https://github.com/user-attachments/assets/68d1304d-ffd5-424e-bc60-ca63d176ba64" />

**Comment by phorcys420** [comment]
Hello @femto, it seems like your machine does not work properly with try.coder.app, this is fine.
Try setting `CODER_ACCESS_URL` to `http://127.0.0.1:3000` and see if it works.
